### PR TITLE
Add State of Our Networks Recap

### PR DESCRIPTION
**Pull request by dcwalk**

### CODE REVIEW

**Comment by dcwalk** [comment]
PLEASE PLEASE review for typos!

File: meeting_notes/20170114_state-of-our-networks.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Finally, during the afternoon discussions EtherMesh and MeshISP covered a range of policy and regulatory concerns, spending time addressing how new initiative fit in Canada alongside robust networks. Toronto Community Networks discussion remained rooed in the need to _know your community_ as a first step in any initiative. Finally, Toronto Mesh considered the asymmetry of power and production in our roles on the Internet, looking to build out a "Help your Neighbour" initiative to build momentum toward joining a mesh network.
```
Comment (benhylau) [inline meeting_notes/20170114_state-of-our-networks.md]: `s/initiative/initiatives`
`s/remained rooed/remain rooted`?

File: meeting_notes/20170114_state-of-our-networks.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+[Pictures](https://www.flickr.com/photos/glennmcknight/albums/72157675518024343) and [audio](https://soundcloud.com/gmcknight/mesh-toronto-panel-discussion) from the day are online.
```
Comment (benhylau) [inline meeting_notes/20170114_state-of-our-networks.md]: You probably know already but the soundcloud is empty.

**Comment by benhylau** [comment]
@dcwalk Thanks for writing this up! I guess we will link this from the timeline too?

**Comment by dcwalk** [comment]
Yeah I knew, I was really hoping the sound would get resolved, no dice from Glenn

**Pull request merged**